packages/asset-uploader/src/utils/format-asset.ts

```diff
@@ -19,13 +19,15 @@ export const formatAsset = (asset: DbAsset): Asset => {
   if (isFont) {
     return {
       ...base,
+      createdAt: base.createdAt.toISOString(),
```

Comment:
is this really the only way we can deal with dates? its going to be everywhere? any remix discussion on this?

Reply:
It's json. Remix just added proper type checking. I'd say it's on prisma to fix but I saw in issue they are not gonna change this. So I guess we either need layer between loaders and prisma or add these hacks.

Reply:
yeah, the place where we convert Date to json is the place where we have to do it and that's basically loaders

Reply:
The issue is that we reuse code for server and client. So I did conversion earlier to make all parts work.

Reply:
https://github.com/Rich-Harris/devalue

sveltekit switched on this